client/verta/verta/_internal_utils/_pip_requirements_utils.py

```diff
@@ -237,9 +235,13 @@ def set_version_pins(requirements):
                 ver = mod.__version__
             except (ImportError, AttributeError):
                 # fall back to checking pip
-                try:
-                    ver = pip_pkg_vers[req]
-                except KeyError:
+                req_with_dash = req.replace("_", "-")
+                req_with_underscore = req.replace("_", "-")
```

Comment:
What's the difference between req_with_dash and req_with_underscore 🤔 The right-hand side of these two statements look the same to me.

Reply:
Bad copy and paste!

Reply:
Oh I think I understand the logic now.